front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -95,25 +93,26 @@ const RightColumnDesktop = ({
           background={colors.background}
           mb="12px"
         >
-          {showGreyBox && (
-            <>
-              <StyledReactionControl
-                styleType="shadow"
-                ideaId={ideaId}
-                size="4"
-              />
-              <Box pb="23px" mb="23px" borderBottom="solid 1px #ccc">
-                {participationContext &&
-                  ideaIsInParticipationContext &&
-                  votingConfig?.getIdeaPageVoteInput({
-                    ideaId,
-                    participationContext,
-                    compact: false,
-                  })}
-              </Box>
-              {commentingEnabled && <Buttons />}
-            </>
-          )}
+          {ideaIsInParticipationContext ||
+            (commentingEnabled && (
+              <>
+                <StyledReactionControl
+                  styleType="shadow"
+                  ideaId={ideaId}
+                  size="4"
+                />
+                <Box pb="23px" mb="23px" borderBottom="solid 1px #ccc">
+                  {participationContext &&
+                    ideaIsInParticipationContext &&
+                    votingConfig?.getIdeaPageVoteInput({
+                      ideaId,
+                      participationContext,
+                      compact: false,
+                    })}
+                </Box>
+                {commentingEnabled && <Buttons />}
```

Comment:
I know this was here before your changes - but "Buttons" makes it really unclear what it is. Do you think we could rename that component so it's more clear? :)